test/common_utils.py

```diff
@@ -209,6 +209,26 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def skipIfCompiledWithoutNumpy(fn):
+    # Even if the numpy module is present, if `USE_NUMPY=0` is used during the
+    # build, numpy tests will fail
+    numpy_support = TEST_NUMPY
+    if numpy_support:
+        try:
+            # The numpy module is present, verify that PyTorch is compiled with
+            # numpy support
+            torch.from_numpy(numpy.array([2, 2]))
```

Comment:
Can we not use `TEST_NUMPY` instead ?

Reply:
Test numpy only checks that `import numpy` works, not that torch was compiled with numpy support